amazing_new_feature.py

```diff
@@ -0,0 +1 @@
+print("This is an amazing new feature!")
```

Comment:
This needs to be wrapped within the API class so that the connection is closed when the request completes.